packages/material-ui/src/TableCell/TableCell.js

```diff
@@ -142,6 +142,9 @@ TableCell.propTypes = {
   component: PropTypes.oneOfType([PropTypes.string, PropTypes.func, PropTypes.object]),
   /**
    * If `true`, content will align to the right.
+   *
+   * Monetary or generally number fields should be right aligned as that allows
```

Comment:
https://ux.stackexchange.com/questions/24066/what-is-the-best-practice-for-data-table-cell-content-alignment/24073#24073?newreg=16f407d305f440559c7e1850edda32c2